File: Code/6-2/4.c
#include <stdio.h>

int main()
{
	int numOne, numTwo, numThree =0;
	int min, max =0;

	scanf("%d %d %d", &numOne, &numTwo, &numThree);
	min = numOne;
	max = numOne;

	if(numTwo <= numOne && numTwo <= numThree)
	{
		min = numTwo;
	}
	else if(numThree <=numOne && numThree <= numTwo)
	{
		min = numThree;
	}

	if(numOne <= numTwo && numThree <= numTwo)
	{
		max = numTwo;
	}
	else if(numOne <= numThree && numTwo <= numThree)
	{
		max = numThree;
	}

	printf("min: %d\n", min);
	printf("max: %d\n", max);

	return 0;
	

}

File: Code/6-2/4.py
numOne, numTwo, numThree = input().split()

numOne = int(numOne)
numTwo = int(numTwo)
numThree = int(numThree)
min = numOne
max = numOne

if(numTwo <= numOne and numTwo <= numThree):
    min = numTwo
elif(numThree <=numOne and numThree <= numTwo):
    min = numThree

if(numTwo >= numOne and numTwo >= numThree):
    max = numTwo
elif(numThree >= numOne and numThree >= numTwo):
    max = numThree

print("min:",min)
print("max:",max)